search-app/src/cmr/search/api/tags.clj

```diff
@@ -20,10 +20,17 @@
   [headers]
   (mt/extract-header-mime-type #{mt/json} headers "content-type" true))
 
-(defn- tag-api-response
+(defn- has-error?
+  "Whether there's an error in [data]."
+  [data]
+  (some (fn [element] (contains? element :errors)) data))
```

Comment:
change this to `(some #(contains? % :errors) data)`

Reply:
resolved